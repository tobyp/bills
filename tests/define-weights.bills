#!./bills
A	X.	1.0

%define X Y2Z

A	X.	3.0
