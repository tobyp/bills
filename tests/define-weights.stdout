todo: X pays 1.0 to A
todo: Y pays 1.0 to A
todo: Z pays 2.0 to A
